web/server/vue-cli/e2e/specs/reports.js

```diff
@@ -279,9 +279,15 @@ module.exports = {
     reportPage.expect.section(dateDialog).to.be.visible.before(5000);
     reportPage.expect.element("@overlay").to.be.visible.before(5000);
 
-    dateDialog
```

Comment:
Please leave this code block here commented out with the note that that this would have been the ideal way, but due to the overlay it was unreliable and maybe a TODO, to change this back afte Vue3 update.

You don't need this comment at all the other places, the simple removal can remain.